clouddq/main.py

```diff
@@ -211,6 +211,14 @@
     is_flag=True,
     default=False,
 )
+@click.option(
+    "--num_bigquery_threads",
```

Comment:
nit: "BigQuery Threads" is a little confusing to me. Maybe just simply call it "concurrency" or "num_threads"?

Reply:
incorporated.